client/web/src/user/settings/codeHosts/CodeHostItem.tsx

```diff
@@ -33,6 +33,7 @@ interface CodeHostItemProps {
     onDidAdd?: (service: ListExternalServiceFields) => void
     onDidRemove: () => void
     onDidError: (error: ErrorLike) => void
+    useGitHubApp: boolean
```

Comment:
Defining this as optional means we can only specify it when needed.
```suggestion
    useGitHubApp?: boolean
```

Reply:
Ah perfect, yeah optional with default behaviour changing nothing is better.